### PR TITLE
Add Incidents Log to 'About the Regulations'

### PR DESCRIPTION
**Pull request by Ivan-Brigidano**
@thewca/wrc-team are you okay with the changes?
I also changed the indentation of some lines to make it more consistent.

### CODE REVIEW

**Comment by Rouxles** [comment]
"Important decisions about incidents that happened in competitions that were resolved by the WCA Regulations Committee are found in the Incidents Log" sounds better, but most definitely isn't perfect. I'll try to think of something better possibly.

**Comment by mirandapedro** [comment]
Looks good to me!!!

**Comment by Ivan-Brigidano** [comment]
@thewca/software-team all WRC members who replied agree with the changes. So you can consider that it has the approval of the Committee.

File: WcaOnRails/app/views/regulations/about/index.html.erb [hunk 1]
```diff
@@ -27,7 +28,7 @@
       <h3 class="panel-title"><%= t('about_regulations.panel3.title') %></h3>
     </div>
     <div class="panel-body">
-        <%= t('about_regulations.panel3.content1_html') %>
+      <%= t('about_regulations.panel3.content1_html') %>
```
Comment (viroulep) [inline WcaOnRails/app/views/regulations/about/index.html.erb:31]: Thanks for fixing the indentation!

File: WcaOnRails/config/locales/en.yml [hunk 2]
```diff
@@ -2165,4 +2166,4 @@ en:
       apply_to_host: Apply to host
     other_suggestions:
       title: Other suggestions?
-      body_html: 'Do you have other suggestions for Instagram or other social media content? <a href="https://www.worldcubeassociation.org/contact/website">Let us know</a>!'
\ No newline at end of file
+      body_html: 'Do you have other suggestions for Instagram or other social media content? <a href="https://www.worldcubeassociation.org/contact/website">Let us know</a>!'
```
Comment (viroulep) [inline WcaOnRails/config/locales/en.yml:2169]: I'd expect most text editors to put a `\n` character at the end of each line, have you configured yours in a specific way?
Except for file format which don't support it (eg: binary files), people usually always have a newline character at end of line.
Most command line tools (cat, head, tail, more, wc, etc...) have their output kind of broken if you don't (for instance the shell prompt ends up on the same line as the last line of the output), and it's also how the posix standard defines a line (you can check this [SO question](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) for even more details).

Comment (viroulep) [reply WcaOnRails/config/locales/en.yml:2169]: I'm dumb, I read the diff the other way around -_-
But I don't understand how this shows up, the missing newline seems to have been fixed in master already...
Anyway, will squash and merge!

**Comment by viroulep** [comment]
Deployed!

**Pull request merged**